dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java

```diff
@@ -197,10 +197,10 @@ private void processQuery(String filter, String sortCol, String sortOrder, Crite
             predicates.add(cb.and(// get published workflows
                 cb.isTrue(entry.get("isPublished")),
                 // ensure we deal with null values and then do like queries on those non-null values
-                cb.or(cb.and(cb.isNotNull(entry.get(nameName)), cb.like(entry.get(nameName), "%" + filter + "%")), //
-                    cb.and(cb.isNotNull(entry.get("author")), cb.like(entry.get("author"), "%" + filter + "%")), //
-                    cb.and(cb.isNotNull(entry.get(repoName)), cb.like(entry.get(repoName), "%" + filter + "%")), //
-                    cb.and(cb.isNotNull(entry.get(orgName)), cb.like(entry.get(orgName), "%" + filter + "%")))));
+                cb.or(cb.and(cb.isNotNull(entry.get(nameName)), cb.like(cb.upper(entry.get(nameName)), "%" + filter.toUpperCase() + "%")), //
```

Comment:
This [StackOverflow page](https://stackoverflow.com/questions/4580285/jpa2-case-insensitive-like-matching-anywhere) points to the possibility of upper() behaving differently in Java and the DB for some edge cases.

I don't think it's worth worrying about for now, but mentioning it just to keep it on the radar.